toponetx/algorithms/spectrum.py

```diff
@@ -255,7 +256,27 @@ def simplicial_complex_hodge_laplacian_spectrum(
     return laplacian_spectrum(SC.hodge_laplacian_matrix(rank=rank))
 
 
-def cell_complex_adjacency_spectrum(CC: CellComplex, rank):
+def path_complex_hodge_laplacian_spectrum(
+    PC: PathComplex, rank: int, weight: str = "weight"
+):
+    """Return eigenvalues of the Laplacian of PC.
+
+    Parameters
+    ----------
+    PC : PathComplex
+    rank : int
+    weight : str or None, optional (default='weight')
+        If None, then each cell has weight 1.
+
+    Returns
+    -------
+    evals : NumPy array
```

Comment:
Single return values do not have a name according to NumPy's documentation style.

```suggestion
    np.ndarray
```